modules/persistent-storage-local-create-cr.adoc

```diff
@@ -46,23 +46,25 @@ spec:
           - ip-10-0-164-33
   storageClassDevices:
     - storageClassName: "local-sc" <3>
-      volumeMode: Filesystem <4>
-      fsType: xfs <5>
-      devicePaths: <6>
-        - /path/to/device <7>
+      forceWipeDevicesAndDestroyAllData: false <4>
+      volumeMode: Filesystem <5>
+      fsType: xfs <6>
+      devicePaths: <7>
+        - /path/to/device <8>
 ----
 <1> The namespace where the Local Storage Operator is installed.
 <2> Optional: A node selector containing a list of nodes where the local storage volumes are attached. This example uses the node hostnames, obtained from `oc get node`. If a value is not defined, then the Local Storage Operator will attempt to find matching disks on all available nodes.
 <3> The name of the storage class to use when creating persistent volume objects. The Local Storage Operator automatically creates the storage class if it does not exist. Be sure to use a storage class that uniquely identifies this set of local volumes.
-<4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
+<4> This setting instructs Local Storage Operator on whether or not to erase the disk before trying to provision it. The default is "false" (the disk is not erased prior to provisioning). Setting `forceWipeDevicesAndDestroyAllData` to "true" can be useful in scenarios where previous data can remain on disks that need to be re-used. In these scenarios, setting this field to true eliminates the need for administrators to erase the disks manually. Such cases can include single-node OpenShift (SNO) cluster environments where a node can be redeployed multiple times or when using OpenShift Container Storage (OCS), where previous data can remain on the disks planned to be consumed as object storage devices (OSDs).
```

Comment:
we need to be more precise here. The wipe option calls wipefs which deletes the partition tables, it does not erase data per se. Technically data remains on the disks but it is ready to be "overwritten". For security transparency this should be explicitly mentioned. 

Also OCS does no exist anymore, it's called ODF now; openshift data foundation

cc @mpatlasov

Reply:
Agreed, we could borrow some language from wipefs man-page:

> wipefs can erase filesystem, raid or partition-table signatures (magic strings) from the specified device to make the signatures invisible for libblkid. wipefs does not erase the filesystem itself nor any other data from the device.

Three key points to highlight: 1) signatures (magic strings) are removed; 2) this makes disk ready to use for LSO provisioning; 3) no other data (other than signatures) erased.

Reply:
Fixed. Please reivew.